javaslang/src-gen/main/java/javaslang/CheckedFunction0.java

```diff
@@ -93,6 +93,18 @@
     }
 
     /**
+     * Narrows the given {@code CheckedFunction0<? extends R>} to {@code CheckedFunction0<R>}
+     *
+     * @param wideFunction A {@code CheckedFunction0}
+     * @param <R> return type
+     * @return the given {@code wideFunction} instance as narrowed type {@code CheckedFunction0<R>}
+     */
+    @SuppressWarnings("unchecked")
+    static <R> CheckedFunction0<R> narrow(CheckedFunction0<? extends R> wideFunction) {
```

Comment:
For Functions / CheckedFunctions we use per default `f` as identifier. However, that's merely the only abbreviation. Predicate and Supplier are named _predicate_ and _supplier_. Consumer is often called _action_ throughout the codebase.

(Changing `wideFunction` to `f` should be a no-brainer because it is generated code.)